### PR TITLE
OSDOCS#8947: Creating new section for capabilities and layered products in the web console

### PR DESCRIPTION
**Pull request by opayne1**
Working with RHTAP(dance) writers to integrate content into the web console section for visibility. This PR as it sits now is only a rough outline for planning and feedback purposes. More will be added over the course of documentation development for the 4.15 release. All feedback welcome.

[Edit] this has gown to include pipelines and serverless for this release but ultimate goal is to make this a resource "landing page" for all things you can do with the web console.

Version(s):
4.15+

Issue:
RHDH and the new section:
https://issues.redhat.com/browse/OSDOCS-8947
https://issues.redhat.com/browse/OSDOCS-9371

Pipelines/Serverless:
https://issues.redhat.com/browse/OSDOCS-7534
https://issues.redhat.com/browse/OSDOCS-8944

Link to docs preview:
https://70141--ocpdocs-pr.netlify.app/openshift-enterprise/latest/web_console/capabilities_layered-products/capabilities-web-console

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -798,6 +798,13 @@ Topics:
 - Name: Creating quick start tutorials
   File: creating-quick-start-tutorials
   Distros: openshift-enterprise,openshift-origin
+- Name: Capabilities and layered products available for the OpenShift Container Platform web console
```
Comment (opayne1) [inline _topic_maps/_topic_map.yml:801]: Update to a variation of 

```suggestion
- Name: Capabilities and layered products
```
Looks too long in the previews.

File: _topic_maps/_topic_map.yml [hunk 2]
```diff
@@ -798,6 +798,13 @@ Topics:
...
+  Dir: capabilities_layered-products
+  Topics:
+  - Name: Additional capabilities available for the OpenShift Container Platform web console
```
Comment (opayne1) [inline _topic_maps/_topic_map.yml:804]:
```suggestion
  - Name: Optional capabilities available for the OpenShift Container Platform web console
```

File: web_console/capabilities_layered-products/layered-products-web-console.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="layered-products-web-console"]
+= Layered products available for the web console
+include::_attributes/common-attributes.adoc[]
+:context: layered-products-web-console
+
+toc::[]
+
+You can further customize the {product-title} web console by adding additinal capabilities to your existing workflows.
```
Comment (vikram-raj) [inline web_console/capabilities_layered-products/layered-products-web-console.adoc:9]:
```suggestion
You can further customize the {product-title} web console by adding additional capabilities to your existing workflows.
```

File: web_console/capabilities_layered-products/capabilities-web-console.adoc [hunk 4]
```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="capabilities-web-console"]
+= Optional capabilities in the web console
+include::_attributes/common-attributes.adoc[]
+:context: capabilities-web-console
+
+toc::[]
+
+You can further customize the {product-title} web console by adding additinal capabilities to your existing workflows.
```
Comment (vikram-raj) [inline web_console/capabilities_layered-products/capabilities-web-console.adoc:9]:
```suggestion
You can further customize the {product-title} web console by adding additional capabilities to your existing workflows.
```

**Review by jhadvig — COMMENTED** [review]
We are refering to these modules as `web consoles` which could make an impression that they are a standalone consoles, rather then part of the OCP console.

**Comment by opayne1** [comment]
@XiyunZhao could you help with a QE review of this PR? Thanks!

**Comment by XiyunZhao** [comment]
The doc looks good to me, links, content, and parameters work as expected
/label qe-approved

**Review by tmalove — APPROVED** [review]

**Comment by Preeticp** [comment]
@rh-max @mramendi @kaldesai I hope you all are aware of this PR. PTAL and reach out to @opayne1 in case anything needs to be updated.

File: modules/pipelines-web-console.adoc [hunk 5]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * layered-products-web-console.adc
+
+:_mod-docs-content-type: CONCEPT
+[id="pipelines-web-console_{context}"]
+= {pipelines-title} in the web console
+
+{pipelines-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. You can use the OperatorHub in the {product-title} web console to install the {pipelines-title} Operator to create and modify pipeline objects.
```
Comment (mramendi) [inline modules/pipelines-web-console.adoc:9]: The double "to" does not work very well in my view. Yes, you can use OperatorHub in the web console to install Popelines. Then you can use the Pipelines view of the Developer web console to create pipelines and to interact with pipelines.

File: modules/rhdh-web-console.adoc [hunk 6]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * layered-products-web-console.adc
+
+:_mod-docs-content-type: CONCEPT
+[id="rhdh-web-console_{context}"]
+= {rh-dev-hub} in the {product-title} web console
+
+The {rh-dev-hub} is a platform you can use for building developer portals with helpful templates to simplify the development process. Quick starts are available for you to learn more about how to install and use the developer hub.
```
Comment (hmanwani-rh) [inline modules/rhdh-web-console.adoc:9]: Is there a specific reason that we mentioned templates in the above sentence? Developer Hub also uses plugins to build developer portals and simplify development. I'd suggest changing the sentence to:

The Red Hat Developer Hub is an open platform for building developer portals, containing a supported and opinionated framework. You can use the quick starts to learn more about how to install and use the Developer Hub.

**Review by hmanwani-rh — COMMENTED** [review]
I've added a comment below. Please feel free to ask any questions you may have. Additionally, should we include links to the RHDH documentation here?

File: modules/rhdh-web-console.adoc (hunk 6 above)
Comment (opayne1) [reply modules/rhdh-web-console.adoc:9]: I got templates from the first paragraph here https://developers.redhat.com/rhdh if that is not the case we should find out who manages that page and update there too.

Comment (opayne1) [reply modules/rhdh-web-console.adoc:9]: Link to RHDH docs is below this entry on line 30

Comment (opayne1) [reply modules/rhdh-web-console.adoc:9]: 👍 will address

**Pull request merged**